tokio/src/sync/mpsc/bounded.rs

```diff
@@ -367,6 +367,22 @@ impl<T> Receiver<T> {
         self.chan.close();
     }
 
+    /// Checks if the channel has been closed. This happens when the
+    /// [`Receiver::close`] method is called.
+    ///
+    /// [`Receiver::close`]: crate::sync::mpsc::Receiver::close
+    ///
```

Comment:
Add an `# Examples` header before the example.

It would probably also be good to add `use tokio::sync::mpsc;` at the top of the example as that's done in **almost** all the other examples (the noteable exception is of course `Sender::is_closed`).